### DISCUSSION TITLE
What About Vanguard? (The valorant anti-cheat

### DISCUSSION THREAD

**Discussion by RentyP** [wasiejen/Free-Snap-Tap#31] [Q&A]
i just wanna know if i use this program on valorant it will get me a ban or smth, im just confused

**Comment by wasiejen** [comment]
Welcome @RentyP ,

like every other script that is used to automate some things in competitive games it can be seen as cheating. I personally think that this script only offers benign forms of automation without any interaction with the game. It just optimizes your input or makes input of some sequences of inputs easier.

And like any other such tool it can lead to consequences in a professional competitive setting. It might simply not work correctly or it might be recognized and you could can kicked. 

I personally think that is about the worst that can happen with this script, but there is never a guarantee for it. Rules might change and it may get bannable.

And honestly speaking, I do not even know if it works correctly with Vanguard. :-D 

I program this more for the coding experience than actually to circumvent any Anti-Cheat (AC) - so testing if it is compatible with various ACs is up to everyone who uses it.
But I will gladly listen to any Feedback that comes my way and see if I can find solutions for eventual problems. :-D
